hyperspy/signal.py

```diff
@@ -4641,12 +4642,13 @@ def map(
         function,
         show_progressbar=None,
         parallel=None,
-        max_workers=None,
         inplace=True,
         ragged=None,
         output_signal_size=None,
         output_dtype=None,
-        **kwargs
+        lazy_result=None,
```

Comment:
To be consistent with `output_dtype` and other, use `lazy_output` instead of `lazy_result`?

Reply:
I've been using `lazy_result` for this in `pyxem`, based on @francisco-dlp suggestion a while back (https://github.com/hyperspy/hyperspy/issues/1840#issuecomment-367296689).

So ideally it would be nice to keep that consistent in `s.map` as well.

Reply:
ok, I changed the argument from `lazy_result` to `lazy_output`.

This might lead to changing the argument name in pyxem as well.